seedvault/src/main/java/com/solanamobile/seedvault/SigningRequest.java

```diff
@@ -44,8 +44,19 @@ public SigningRequest(@NonNull byte[] payload, @NonNull List<Uri> requestedSigna
     }
 
     protected SigningRequest(Parcel in) {
-        payload = in.createByteArray();
```

Comment:
`createByteArray` and `createTypedArrayList` can return null, but class fields are marked `@NonNull`